Support syscoin

tomatoskittles opened the pull request:
We would like to add Syscoin and Syscoin Testnet. Please let me know if there is anything else you would like me to do.

ligi commented on _data/chains/eip155-57.json:
```diff
@@ -0,0 +1,24 @@
+    "https://web3.syscoin.org",
+    "wss://web3.syscoin.org"
+  ],
+  "faucets": ["https://faucet.syscoin.org"]
```
```suggestion
  "faucets": ["https://faucet.syscoin.org"],
```

ligi commented on _data/chains/eip155-5700.json:
```diff
@@ -0,0 +1,22 @@
+  "rpc": [
+    "https://tanenbaum.io/api/eth-rpc"
+  ],
+  "faucets": ["https://faucet.tanenbaum.io"]
```
```suggestion
  "faucets": ["https://faucet.tanenbaum.io"],
```

ligi reviewed (changes requested):
there are 2 commas missing - also the explorers need the `standard` field

tomatoskittles commented:
> there are 2 commas missing - also the explorers need the `standard` field

Thanks for the review.

I have fixed the comma related syntax error, and I have also removed the explorers. We are using Blockscout, which I don't believe is EIP3091 compliant. xDai also uses Blockscout, and their entry doesn't have explorers either, so I figured this was a good approach.

Hopefully it is good, now.

ligi reviewed (changes requested):
short name must be unique - but found `sys` more than once

tomatoskittles commented:
> short name must be unique - but found `sys` more than once

I've changed the testnet shortname, to honour the uniqueness requirement.
